code/modules/antagonists/mind_flayer/powers/flayer_mobility_powers.dm

```diff
@@ -0,0 +1,127 @@
+/datum/spell/ethereal_jaunt/proc/create_jaunt_holder(turf/mobloc, mob/living/target)
+	return new jaunt_type_path(mobloc)
+
+//Basically shadow anchor, but the entry and exit point must be computers. I'm not in your walls I'm in your PC
+/datum/spell/flayer/computer_recall
+	name = "Traceroute"
+	desc = "Cast once to mark a computer, then cast this next to a different computer to recall yourself back to the first. Alt click to check your current mark."
+	base_cooldown = 60 SECONDS
+	action_icon_state = "pd_cablehop"
+	upgrade_info = "Halve the time it takes to recharge."
+	power_type = FLAYER_PURCHASABLE_POWER
+	category = CATEGORY_INTRUDER
+	centcom_cancast = FALSE
+	var/obj/machinery/computer/marked_computer = null
```

Comment:
Move this to the bottom of the typedef, please, and make sure it's cleared properly on destroy.